spec/models/letter_opener_web/letter_spec.rb

```diff
@@ -1,7 +1,7 @@
 # frozen_string_literal: true
 
 describe LetterOpenerWeb::Letter do
-  let(:location) { File.expand_path('../../tmp', __dir__) }
+  let(:location) { Pathname.new(__dir__).join('..', '..', 'tmp').cleanpath }
```

Comment:
thought: `LetterOpenerWeb.config.letters_location` is actually an instance of `Pathname` so we can leverage its powers 🙂